### PR TITLE
Slightly improve no-extend-struct-new wording and add code example

### PR DESCRIPTION
**Pull request by mmozuras**
@bbatsov

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2301,11 +2301,21 @@ condition](#safe-assignment-in-condition).
   ````
 
 * <a name="no-extend-struct-new"></a>
-  Don't extend a `Struct.new` - it already is a new class. Extending it
-  introduces a superfluous class level and may also introduce weird errors if
-  the file is required multiple times.
+  Don't extend instance initialized by `Struct.new`. Extending it introduces
```
Comment (bbatsov) [inline README.md]: an instance

File: README.md [hunk 2]
```diff
@@ -2301,11 +2301,21 @@ condition](#safe-assignment-in-condition).
...
 
+  ```Ruby
+  # bad
+  class Person < Struct.new(:first_name, :last_name)
+  end
+
+  # good
+  Person = Struct.new(:first_name, :last_name) do
+  end
+  ````
```
Comment (bbatsov) [inline README.md]: Those examples are slightly different. You should either add `do/end` to the first `Struct.new` or remove it from the second.

File: README.md (hunk 1 above)
Comment (mmozuras) [inline README.md]: @bbatsov :+1:

File: README.md (hunk 2 above)
Comment (mmozuras) [inline README.md]: @bbatsov removed `do/end`.

**Pull request merged**